docs/administration-guide/job-tracker.md

```diff
@@ -0,0 +1,139 @@
+# Job Tracker
+
+This section covers how to configure your Gaffer graph to allow jobs to be executed.
+
+'Job' refers to an operation chain with a long-running sequence of operations that is executed
+asynchronously. 
+Jobs are, therefore, useful for operation chains which take a long time to return results or if 
+you want to cache the results of your operation chain. 
+
+Jobs are only available when the job tracker has been enabled. 
+The job tracker can then be used to store the status of current and historic jobs 
+while a 'cache' is used to store any job results. 
+
+## Configuration
+
+By default, the job tracker is disabled. 
+It can be enabled by setting its associated [store property](../administration-guide/gaffer-stores/store-guide#store-properties) to true.
+There must also be a [cache configured](../administration-guide/gaffer-stores/store-guide.md#caches) for the job
+tracker to work.
+
+### Configuring the Results Cache
+To store any results using the job tracker, a `GafferResultCache` needs to enabled. 
+This job results cache can then store your results using operations and operation handlers 
+(these are disabled by default). 
+Note that this is not the same as your initial cache, as this results 'cache' is simply a second Gaffer graph
+which will store your results.
+
+To store results using the job tracker a few elements must be set up. 
+Firstly, you need to add two operations to an [operations declarations JSON file](../development-guide/example-deployment/project-setup.md#operations-declarations). 
```

Comment:
Might be better to link to the configuration page instead https://gchq.github.io/gaffer-doc/latest/administration-guide/gaffer-config/config/#operationsdeclarationsjson